plugin/src/main/java/org/opensearch/ml/action/models/GetModelTransportAction.java

```diff
@@ -55,7 +57,8 @@ public GetModelTransportAction(
     protected void doExecute(Task task, ActionRequest request, ActionListener<MLModelGetResponse> actionListener) {
         MLModelGetRequest mlModelGetRequest = MLModelGetRequest.fromActionRequest(request);
         String modelId = mlModelGetRequest.getModelId();
-        GetRequest getRequest = new GetRequest(ML_MODEL_INDEX).id(modelId);
+        FetchSourceContext fetchSourceContext = fetchSourceContext(mlModelGetRequest.isReturnContent());
```

Comment:
Try this ?
```
        GetRequest getRequest = new GetRequest();
        FetchSourceContext featchContext = new FetchSourceContext(true, includes, excludes);
        getRequest.index(ML_MODEL_INDEX).id(modelId).fetchSourceContext(featchContext);
```

Reply:
This logic is included in the util function fetchSourceContext(...). I realized this function name should be changed.